tests/features/search/search.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+const bookETLDataEntity = 'Book_ETL_aqa';
+const specialCharactersDataEntity = '_AirFlights';
+const numbersDataEntity = '737boeing';
```

Comment:
The first 2 vars look excellent:
```
const entityNameWithAlphabeticChars = 'Book_ETL_aqa';
const entityNameWithSpecialChars = '_!AirFlights';
const numbersDataEntity = '737boeing';
```
@makonakro, could we please also rename the third var too?

So that all these entity names will have a similar naming pattern -> it will be easier to read and understand tests with such var names.

If the proposed name looks longer, then what about `entityNameWithNumbers`?